spec/board_spec.rb

```diff
@@ -0,0 +1,52 @@
+require './lib/board.rb'
+
+RSpec.describe Board do
+  describe '#initialize' do
+
+    let(:board){(1..9).to_a}
+    it 'creates a new board with an array of 1-9 elements ' do
+     expect(board).to eql([1,2,3,4,5,6,7,8,9])
+    end  
+  end 
+
+  
+  describe '#winner' do
+
+    let(:winner){@winner = nil}
+    it 'sets winner value to nil' do 
+      expect(winner).to eql(nil)

```

Comment:
:bug: This test is not connected with the Board Class, that is, commenting the code for Board Class still passes this test.

Let's use `let` to call an instance of the board class instead and test the methods on it.

#### This is true for all 15 `examples` in this test. Please do this for all other occurrences.